pidgin/app.py

```diff
@@ -0,0 +1,113 @@
+import json
+
+import flask
+import requests
+
+
+app = flask.Flask(__name__)
+
+
+# Endpoint to get core metadata as JSON from an object_id.
```

Comment:
In general, comments about the purpose of a function, or explaining the arguments and so forth, should go as a docstring inside the function:
```python
@app.route('/json/<path:object_id>') # 'path' allows the use of '/' in the id
def get_json_metadata(object_id):
    """
    Get core metadata as JSON from an object_id.
    """
```
We use this imperative style for python docstrings, so you write "do this" instead of "this function will do this" (like I used in the example).